DOCUMENTS/POSTFIX_CONF.txt

```diff
@@ -167,6 +208,7 @@ for file in $map_files ; do
 			s/^password =.*/password = $password/ ;
 			s/^hosts =.*/hosts = $hosts/ ;
 			s/^dbname =.*/dbname = $dbname/ ;
+			s/\$myhostname/$myhostname/ ;
```

Comment:
If you use @@myhostname@@ in the config file template, you make your life easier (no need for escaping the $ sign) and also make it clear for people who read POSTFIX_CONF.TXT that it's only a placeholder.

BTW: A note about being @@myhostname@@ (or $myhostname if you insist on the name) being a placeholder would be nice ;-)